vars/perfRegressionParallelPipeline.groovy

```diff
@@ -13,7 +13,7 @@ def call(Map pipelineParams) {
         environment {
             AWS_ACCESS_KEY_ID     = credentials('qa-aws-secret-key-id')
             AWS_SECRET_ACCESS_KEY = credentials('qa-aws-secret-access-key')
-            SCT_GCE_PROJECT = "${params.gce_project}"
+            SCT_GCE_PROJECT = "${params.gce_project ?: ''}"
```

Comment:
how this is related to this PR ?

Reply:
I think this breaks the collection:
```
2:51:18  Traceback (most recent call last):
22:51:18    File "/home/ubuntu/scylla-cluster-tests/./sct.py", line 1861, in <module>
22:51:18      cli.main(prog_name="hydra")
22:51:18    File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
22:51:18      rv = self.invoke(ctx)
22:51:18    File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
22:51:18      return _process_result(sub_ctx.command.invoke(sub_ctx))
22:51:18    File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
22:51:18      return ctx.invoke(self.callback, **ctx.params)
22:51:18    File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
22:51:18      return __callback(*args, **kwargs)
22:51:18    File "/home/ubuntu/scylla-cluster-tests/./sct.py", line 1233, in collect_logs
22:51:18      collected_logs = collector.run()
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/logcollector.py", line 1587, in run
22:51:18      if result := log_collector.collect_logs(local_search_path=local_dir_with_logs):
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/logcollector.py", line 947, in collect_logs
22:51:18      builders = get_builder_by_test_id(self.test_id)
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/common.py", line 2413, in get_builder_by_test_id
22:51:18      builders = list_builders(running=True)
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/common.py", line 2404, in list_builders
22:51:18      gce_builders = get_gce_builders(builder_tag, running=running)
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/common.py", line 2388, in get_gce_builders
22:51:18      gce_builders = list_instances_gce(tags_dict=tags, running=running)
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/common.py", line 1340, in list_instances_gce
22:51:18      instances_client, info = get_gce_compute_instances_client()
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/gce_utils.py", line 61, in get_gce_compute_instances_client
22:51:18      info = KeyStore().get_gcp_credentials()
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/keystore.py", line 56, in get_gcp_credentials
22:51:18      return self.get_json(f"{project}.json")
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/keystore.py", line 42, in get_json
22:51:18      return json.loads(self.get_file_contents(json_file))
22:51:18    File "/home/ubuntu/scylla-cluster-tests/sdcm/keystore.py", line 38, in get_file_contents
22:51:18      return obj.get()["Body"].read()
22:51:18    File "/usr/local/lib/python3.10/site-packages/boto3/resources/factory.py", line 580, in do_action
22:51:18      response = action(self, *args, **kwargs)
22:51:18    File "/usr/local/lib/python3.10/site-packages/boto3/resources/action.py", line 88, in __call__
22:51:18      response = getattr(parent.meta.client, operation_name)(*args, **params)
22:51:18    File "/usr/local/lib/python3.10/site-packages/botocore/client.py", line 534, in _api_call
22:51:18      return self._make_api_call(operation_name, kwargs)
22:51:18    File "/usr/local/lib/python3.10/site-packages/botocore/client.py", line 976, in _make_api_call
22:51:18      raise error_class(parsed_response, operation_name)
22:51:18  botocore.errorfactory.NoSuchKey: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
```

Reply:
this change was exactly a try to fix that issue - faced on master not only for this job.
Somehow `null` string is passed to `SCT_GCE_PROJECT=null` and I don't understand why.
This is regression introduced while working with different gce project in Z3 efforts.
I need to find other way with the next run as my test failed due https://github.com/scylladb/cql-stress/pull/86#issuecomment-2204524664

Reply:
I think it's coming from here:
```
"SCT_GCE_PROJECT=${env.SCT_GCE_PROJECT}"
```

I think jenkins env is translate empty to null

Reply:
I switched it to `params.gce_project` and worked